lib/Settings/Admin.php

```diff
@@ -0,0 +1,106 @@
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Notifications\Settings;
+
+use OCA\Notifications\AppInfo\Application;
+use OCA\Notifications\Model\Settings;
+use OCA\Notifications\Model\SettingsMapper;
+use OCP\AppFramework\Db\DoesNotExistException;
+use OCP\AppFramework\Http\TemplateResponse;
+use OCP\AppFramework\Services\IInitialState;
+use OCP\IConfig;
+use OCP\IL10N;
+use OCP\IUser;
+use OCP\Settings\ISettings;
+use OCP\IUserSession;
+use OCP\Util;
+
+class Admin implements ISettings {
+    protected IConfig $config;
+    protected IL10N $l10n;
+    private SettingsMapper$settingsMapper;
+    private IUserSession $session;
+    private IInitialState $initialState;
+
+    public function __construct(IConfig        $config,
+                                IL10N          $l10n,
+                                IUserSession   $session,
+                                SettingsMapper $settingsMapper,
+                                IInitialState  $initialState)
+    {
+        $this->config = $config;
+        $this->l10n = $l10n;
+        $this->settingsMapper = $settingsMapper;
+        $this->session = $session;
+        $this->initialState = $initialState;
+    }
+
+    public function getForm(): TemplateResponse {
+        Util::addScript('notifications', 'notifications-adminSettings');
+
+        $default_sound_notification = $this->config->getAppValue(Application::APP_ID, 'sound_notification') === 'yes' ? 'yes' : 'no';
+        $default_sound_talk = $this->config->getAppValue(Application::APP_ID, 'sound_talk') === 'yes' ? 'yes' : 'no';
+        $default_batchtime = $this->config->getAppValue(Application::APP_ID, 'setting_batchtime');
+
+        if ($default_batchtime != Settings::EMAIL_SEND_WEEKLY
+            && $default_batchtime != Settings::EMAIL_SEND_DAILY
+            && $default_batchtime != Settings::EMAIL_SEND_3HOURLY
+            && $default_batchtime != Settings::EMAIL_SEND_HOURLY
+            && $default_batchtime != Settings::EMAIL_SEND_OFF) {
+            $default_batchtime = Settings::EMAIL_SEND_3HOURLY;
+        }
+
+        $this->initialState->provideInitialState('config', [
+            'setting' => 'admin',
+            'setting_batchtime' => $default_batchtime,
+            'sound_notification' => $default_sound_notification === 'yes',
+            'sound_talk' => $default_sound_talk === 'yes',
```

Comment:
We use lowerCamelCase

```suggestion
        $defaultSoundNotification = $this->config->getAppValue(Application::APP_ID, 'sound_notification') === 'yes' ? 'yes' : 'no';
        $defaultSoundTalk = $this->config->getAppValue(Application::APP_ID, 'sound_talk') === 'yes' ? 'yes' : 'no';
        $defaultBatchtime = $this->config->getAppValue(Application::APP_ID, 'setting_batchtime');

        if ($defaultBatchtime != Settings::EMAIL_SEND_WEEKLY
            && $defaultBatchtime != Settings::EMAIL_SEND_DAILY
            && $defaultBatchtime != Settings::EMAIL_SEND_3HOURLY
            && $defaultBatchtime != Settings::EMAIL_SEND_HOURLY
            && $defaultBatchtime != Settings::EMAIL_SEND_OFF) {
            $defaultBatchtime = Settings::EMAIL_SEND_3HOURLY;
        }

        $this->initialState->provideInitialState('config', [
            'setting' => 'admin',
            'setting_batchtime' => $defaultBatchtime,
            'sound_notification' => $defaultSoundNotification === 'yes',
            'sound_talk' => $defaultSoundTalk === 'yes',
```